CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Update helm chart with modern labels, configuration, and templating.
 - Switch from dep to Go modules.
 - Use release.Revision in Helm chart for Helm 3 support.
+- Only replace nodes when ignition changes.
```

Comment:
Not related to this change, just wanted to mention it since it was missed in the previous PR.